hooks/post/31-mongo_syspath.rb

```diff
@@ -0,0 +1,17 @@
+def install_syspaths
+  `rpm -qa | grep rh-mongodb34-syspaths`
+  if $?.success?
+    logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
+  else
+    logger.info 'rh-mongodb34-syspaths not present, installing.'
+    Kafo::Helpers.execute('yum install -y -q rh-mongodb34-syspaths')
+  end
+end
+
+katello = Kafo::Helpers.module_enabled?(@kafo, 'katello')
+foreman_proxy_content = Kafo::Helpers.module_enabled?(@kafo, 'foreman_proxy_content')
+if katello || foreman_proxy_content
+  install_syspaths
+else
+  logger.debug 'Selected scenario is not correct for rh-mongodb34-syspaths install, skipping'
```

Comment:
`not correct` is IMHO incorrect because a user could read and think something is wrong. I'd say `not applicable`.